x-pack/plugins/threat_intelligence/common/types/indicator.ts

```diff
@@ -56,12 +56,25 @@ export enum RawIndicatorFieldId {
  * (reverse of https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/cti/constants.ts#L35)
  */
 export const IndicatorFieldEventEnrichmentMap: { [id: string]: string[] } = {
+  [RawIndicatorFieldId.FileSha256]: ['file.hash.sha256'],
```

Comment:
added a bunch of missing values, as the grid was crashing when trying to display the investigate in timeline button

Reply:
Are we sure that this fixes it for good? What if there is no `threat.indicator.file.hash.*` field on the document. See for example https://github.com/elastic/seceng/issues/4768#issuecomment-1303165947 , their documents have the prefix `threatintel.*` instead of `threat.*` therefore neither of these mappings will work. But I believe we should still handle this gracefully and don't render the Investigate In Timeline action if it is not supported

Reply:
I also think this is a good candidate for 8.5 backporting

Reply:
I think it all depends on how we handle the original bug: if we show the value in the grid with a `-` for example (as you suggested earlier) then this might break. If we don't show the entry in the grid this should be good, as this list matches the list we use in the painless script.

Once we figure out the original bug, I can update this piece of code as well

Reply:
I added a bit more safety to the `use_investigate_in_timeline` hook to avoid issues when a new hash is added to one list and not added to another. Ideally, we let the type checker do that and do not rely on syncing two lists manually